tests/workbenches/notebook-controller/test_data/notebook.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: kubeflow.org/v1
+kind: Notebook
```

Comment:
is there a template for this in the cluster?

Reply:
no, we don't have template for notebooks, the only template we have is hardcoded in odh-dashboard

https://github.com/opendatahub-io/odh-dashboard/blob/eabd833b5f4b0508fe5b06235f4f670fc7e9a0db/backend/src/utils/notebookUtils.ts#L260-L280

Reply:
i like it this way, see no reason to rewrite it

Reply:
ended up rewriting it into python dict